lib/queryShadowDom/index.js

```diff
@@ -0,0 +1,93 @@
+/**
```

Comment:
In this file as well have the param names consistent for dom element
Name it `element` or `node`